Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
+
+/* Add discount code message */
+"Add your discount code bellow:" = "Fügen Sie Ihren Rabatt Code unten hinzu:";
+
+/* Discount */
+"Discount" = "Rabatt";
+
+/* No discounts */
+"No discounts - please redeem a code." = "Keine Rabatte - bitte erlöschen Sie einen Code.";
```

Comment:
Feel free to check out the build - this is actually shown in the popup presented when you tap on ℹ️ icon in the cart summary (discount section).